qemu/tests/iperf_test.py

```diff
@@ -86,6 +87,17 @@ def is_iperf_running(name_pattern, session=None):
                                     ignore_status=True, verbose=False)
         return status == 0
 
+    def rss_test():
+        logging.info('Run the command "netkvm-wmi.cmd rss" to collect statistics')
+        rss_test_cmd = utils_misc.set_winutils_letter(
+                guest_session, params["rss_test_cmd"])
+        output = guest_session.cmd_output(rss_test_cmd)
+        value1 = int(output.splitlines()[1].split()[-4])
+        value2 = int(output.splitlines()[1].split()[-2])
```

Comment:
Could you provide the output of `rss_tes_cmd`? Just want to determine if there is a better way to get value1/value2, otherwise we can let `output = guest_session.cmd_output(rss_test_cmd)splitlines()[1].split()`.

Reply:
its output like as following,What we need is the value of rxErrors and rxMissed to judge whether the execution is successful, I am not sure if there is a more suitable way, could you provide some help？Thanks a lot.
[stdout] Active  DeviceSupport  InstanceName                     rxErrors  rxHits   rxMissed  rxUnclassified
[stdout] TRUE    1              Red Hat VirtIO Ethernet Adapter  0         2109149  0             127

Reply:
A better way is to get the index of `rxErrors` and `rxMissed` in line 1 and use the same index to get their values ​​in line 2 because we cannot be sure whether their order will change later.

But for now, use the current way is okay, let's add a line `rss_statistics = output.splitlines()[1].split()` and get value1 and value2 from it.

Reply:
Hi @PaulYuuu, please help me review it again, thanks a lot.